src/Routing.tsx

```
@@ -1,51 +1,65 @@
-import MigrationAssessmentPage from "./pages/MigrationAssessmentPage";
-import MigrationWizardPage from "./pages/MigrationWizardPage";
+import React, { Suspense } from 'react';
+import { Route as RouterRoute, Routes as RouterRoutes } from 'react-router-dom';
+
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import { InvalidObject } from '@redhat-cloud-services/frontend-components/InvalidObject';
+
+import AssessmentDetailsPage from './pages/AssessmentDetailsPage';
+import MigrationAssessmentPage from './pages/MigrationAssessmentPage';
+import MigrationWizardPage from './pages/MigrationWizardPage';
+import { Provider as DiscoverySourcesProvider } from './migration-wizard/contexts/discovery-sources/Provider';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)